packages/link-bins/src/index.ts

```diff
@@ -120,6 +120,10 @@ async function getPackageBins (
     return []
   }
 
+  if (typeof pkg.bin === 'string' && !pkg.name) {
```

Comment:
I guess we could also check for `R.isEmpty(pkg.bin)`

Reply:
Can we use `!pkg.bin`? `R.isEmpty(pkg.bin)` would return true if it's `undefined`

Reply:
maybe add both. I also mean the case when bin is {}